### ISSUE TITLE
Assistants Open ai.

### ISSUE THREAD

**Issue by ferstpetr777** [n4ze3m/dialoqbase#141]
Nazeem, can you please tell me if the Assistants  will be integrated into the dialogbase program? https://platform.openai.com/docs/assistants/overview
update has been released Open ai

**Comment by n4ze3m** [comment]
Yes, it will be integrated into Dialoqbase, but first, I need to check the current support with Langchain 😃

**Comment by ferstpetr777** [comment]
Great, we're looking forward to it, your app is the best! It will be even better - ahead of the planet.

**Comment by oalmajalists** [comment]
Hope Assistant API integration is coming soon .
